File: HCHDataPlanRoomsData.cs
using System;

namespace BossAdmin
{
    public partial class HCHDataPlanRoomsData
    {
        public partial class SpPlanRoomsGetTableAdapter
        {
            public string ConnectionString
            {
                get
                {
                    return Connection.ConnectionString;
                }
                set
                {
                    Connection.ConnectionString=value;
                }
            }

            private object Connection { get; set; }

        }

        private void HCHDataPlanRoomsData_Initialized(object sender, EventArgs e)
        {

        }
    }
}

File: HCHDataPlanRoomsData.vb
Partial Class HCHDataPlanRoomsData
    Partial Class SpPlanRoomsGetTableAdapter
        Public Property ConnectionString() As String
            Get
                Return Connection.ConnectionString
            End Get
            Set(value As String)
                Connection.ConnectionString = value
            End Set
        End Property

        Private Property Connection As Object

    End Class

    Private Sub HCHDataPlanRoomsData_Initialized(sender As Object, e As EventArgs) Handles Me.Initialized

    End Sub
End Class
